Updated with the Flutter instructions. (#2)
beth-panx/win-student-dev: samples/Flutter/flutter.md

# Build an awesome Flutter app for Windows

Link to video: https://aka.ms/win-dev/student/osu/flutter/get-started-video
Link to blog: https://aka.ms/win-dev/student/osu/flutter/get-started-blog

> You can follow the official documentation [here](https://flutter.dev/docs/get-started/install/windows) but we'll provide all the steps required are prepare your development environment to build Flutter apps for Windows.

Reviewer: ```suggestion
You can follow the official documentation [here](https://flutter.dev/docs/get-started/install/windows) but we will provide all the steps required to prepare your development environment to build Flutter apps for Windows.
```

## Installing Flutter

> In order to build your Windows app for Flutter, you need to have a few things installed on your system.

Reviewer: ```suggestion
In order to build your Flutter app for Windows, you need to have a few things installed on your system.
```

> To install these dependencies, we recommend using winget. Winget requires Windows 10, version 1809 (10.0.17763), or later. Lets start by going to the winget website:

Reviewer: ```suggestion
To install these dependencies, we recommend using [winget](https://docs.microsoft.com/en-us/windows/package-manager/winget/). Winget requires Windows 10, version 1809 (10.0.17763), or later. Lets start by going to the winget website:
```

```winget install -e --id Git.Git```

Next is Visual Studio 2019. If you don't have a professional license for Visual Studio 2019, then you can simply use the Community edition:

```winget install -e --id Microsoft.VisualStudio.Community```

After installing Visual Studio 2019, open the Visual Studio Installer, where we will choose a dependency that needs to be present within VS. If you are having trouble finding it, just press the *Windows* they, then type "**Visual Studio Installer**". With the installer opened, click on *Modify* and make sure that the "**Desktop development with C++**" workload is selected. If it isn't, just select it and click on Modify on the lower right corner of the installer. This will ensure you have everything required to build your Flutter Windows Application. Just as some background, whenever you build your project, the Flutter commands will generate a native project to whichever platform that you are building for, and in the Windows case, it generates a native C++ Win32 application project, that uses Visual Studio 2019 to build your final executable.

> Now, lets install the Visual Studio Code Flutter extension. Open Visual Studio Code, then in the extensions tab (you can use the *Ctrl+Shift+X* shortcut to open it) search for "**Flutter**". When you find it, select it on the list, and click on "**Install**".

Reviewer: ```suggestion
Now, lets install the Visual Studio Code Flutter extension. Open Visual Studio Code, then in the extensions tab (you can use the *Ctrl+Shift+X* shortcut to open it) search for "**Flutter**". When you find it, select it on the list, and click "**Install**".
```
